qiita_db/metadata_template/prep_template.py

```diff
@@ -111,7 +108,7 @@ def create(cls, md_template, raw_data, study, data_type,
 
         # Get a connection handler
         conn_handler = SQLConnectionHandler()
-        queue_name = "CREATE_PREP_TEMPLATE_%d" % raw_data.id
+        queue_name = "CREATE_PREP_TEMPLATE_%d" % study.id
```

Comment:
Agree that is not likely to happen, however it is really easy to make it unique, so fixing because I think it can be a pain to try to debug an issue with this:

```python
queue_name = "CREATE_PREP_TEMPLATE_%d_%d" % (study.id, id(md_template))
```